crates/blockchain/mempool.rs

```diff
@@ -18,21 +18,20 @@ use ethrex_common::{
 use ethrex_storage::error::StoreError;
 use std::collections::HashSet;
 
-// Max number of transactions in the mempool
-const MEMPOOL_MAX_SIZE: usize = 10000; //TODO: Define
-
 #[derive(Debug, Default)]
 pub struct Mempool {
     broadcast_pool: RwLock<HashSet<H256>>,
     transaction_pool: RwLock<HashMap<H256, MempoolTransaction>>,
     blobs_bundle_pool: Mutex<HashMap<H256, BlobsBundle>>,
     txs_by_sender_nonce: RwLock<BTreeMap<(H160, u64), H256>>,
     txs_order: RwLock<Vec<H256>>,
+    max_mempool_size: usize,
 }
 impl Mempool {
-    pub fn new() -> Self {
+    pub fn new(max_mempool_size: usize) -> Self {
         Mempool {
-            txs_order: RwLock::new(Vec::with_capacity(MEMPOOL_MAX_SIZE)),
+            txs_order: RwLock::new(Vec::with_capacity(max_mempool_size)),
```

Comment:
We can probably reserve capacity for `txs_by_sender_nonce` and `transaction_pool` too. That won't really affect performance but can reduce noise when profiling memory, and show us if the mempool is somehow exceeding capacity due to bugs